catalogue.html

```diff
@@ -69,22 +69,29 @@
     </nav>
     <h1>ПЕРФОРАТОРЫ</h1>
     <section>
-      <h3>ПРОИЗВОДИТЕЛИ:</h3>
-      <ul>
-        <li>BOSCH</li>
-        <li>ИНТЕРСКОЛ</li>
-        <li>MAKITA</li>
-        <li>DEWALT</li>
-        <li>HITACHI</li>
-      </ul>
-      <h3>ПИТАНИЕ:</h3>
-      <ul>
-        <li>АККУМУЛЯТОРНЫЕ</li>
-        <li>СЕТЕВЫЕ</li>
-      </ul>
-      <div>
-        <button type="button">ПОКАЗАТЬ</button>
-      </div>
+      <h2 class="visually-hidden">Фильтр</h2>
+      <form class="#" action="#" method="get">
+        <fieldset>
+          <legend>ПРОИЗВОДИТЕЛИ:</legend>
+          <ul>
+            <li><input id="bosch" type="checkbox" name="BOSCH" /><label for="bosch">BOSCH</label></li>
```

Comment:
имена данных формы пишутся маленькими буквами
```suggestion
            <li><input id="bosch" type="checkbox" name="bosch"><label for="bosch">BOSCH</label></li>
```
закрывающий тег не нужен